pkg/validatormanager/validatormanager.go

```diff
@@ -186,6 +216,7 @@ func SetupPoA(
 	aggregatorAllowPrivatePeers bool,
 	aggregatorLogger logging.Logger,
 	validatorManagerAddressStr string,
+	useACP99 bool,
 ) error {
 	return subnet.InitializeProofOfAuthority(
```

Comment:
doesn't have to be InitializeProofOfAuthority anymore. Can just be initialize alidator manager

Reply:
same. another PR be needed to check that PoS is supported on ACP99. For the moment this is
v1.0.0/ACP99 for PoA, v1.0.0 for PoS